cl/audio/migrations/0006_audio_update_sources_noop.py

```diff
@@ -0,0 +1,193 @@
+# Generated by Django 4.2.4 on 2023-10-13 22:01
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        (
+            "audio",
+            "0005_remove_audio_update_or_delete_snapshot_update_and_more",
+        ),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="audio",
+            name="source",
+            field=models.CharField(
+                blank=True,
+                choices=[
+                    ("C", "court website"),
```

Comment:
Why are we doing an audio migration?

Reply:
because it uses the same sources as opinion cluster model and we updated the cluster sources, that's why django generated a new migration